storage_manager_new.py

```diff
@@ -5,184 +5,260 @@ class Storage:
+        items_in_file = File.readlines()
+        File.close()
+        for item in list_of_items_to_delete:
+            items_in_file.remove(item)
             File = open(Storage.database_file, 'w')
+            for line in items_in_file:
+                File.write(line)
             File.close()
-            initialize()
-        else:
-            pass
+        self.clear_self_items_list_and_load_items_from_file()
+
+    def delete_all_items(self):
+        File = open(Storage.database_file, 'w')
+        File.close()
+        self.clear_self_items_list_and_load_items_from_file()
 
 
 class Item:
-    instances = []
 
     def __init__(self, category, type, model):
         self.category = category
         self.type = type
         self.model = model
-        Item.instances.append(self)
+
+    def __str__(self):
+        return f'({self.category})  {self.type} {self.model}'
 
     @classmethod
-    def instance_from_string(cls, string):
-        (category, type, model) = string.split(', ')
-        cls(category, type, model)
+    def create_instance_from_string_and_write_to_database_file(cls, stringified_item):
+        instance = Item.create_instance_from_string(stringified_item)
+        instance.write_to_database_file()
 
     @classmethod
-    def create_instance_from_string_and_write_to_database_file(cls, string):
-        Item.instance_from_string(string)
+    def create_instance_from_string(cls, stringified_item):
+        (category, type, model) = stringified_item.split(', ')
+        return cls(category, type, model)
+
+    def write_to_database_file(self):
         File = open(Storage.database_file, 'a')
-        File.write(string + '\n')
+        File.write(f'{self.category}, {self.type}, {self.model}\n')
         File.close()
 
 
 class Interface:
-    def __init__(self, name, option_1, option_2, option_3, option_4, option_5):
-        self.name = name
-        self.option_1 = option_1
-        self.option_2 = option_2
-        self.option_3 = option_3
-        self.option_4 = option_4
-        self.option_5 = option_5
-
-    def menu_option_list(self):
-        option_list = [self.option_1, self.option_2,
-                       self.option_3, self.option_4, self.option_5]
-        return option_list
-
-    def start_program_interface(self, storage):
-        print('What would you like to do? Type a number.\n')
-        self.show_action_options_and_execute_user_choice(storage)
-
-    def show_action_options_and_execute_user_choice(self, storage):
-        menu = self.create_and_show_option_menu()
-        choice = Interface.get_user_choice(None)
+    option_dict = {1: 'Add item', 2: 'Show all items',
+                   3: 'Show list of items from a certain category or type', 4: 'Delete item or group of items', 5: 'Delete all items'}
+
+    def __init__(self, name, storage):
+        self.storage = storage
+        self.start_program_interface()
+
+    def start_process_of_adding_items(self):
```

Comment:
These `start_process_` methods have a bit misleading names. I know you need some different names from the ones that actually do things in db but `start_process_` suggests it is only part of the task and other part(s) is going one somewhere else while in given method(s) you actually start and finish whole process.